src/CalculatorUITestFramework/WinAppDriver.cs

```diff
@@ -12,6 +12,7 @@ namespace CalculatorUITestFramework
     public sealed class WinAppDriver
     {
         private WindowsDriverLocalService windowsDriverService = null;
+        public MemoryPanel MemoryPanel = new MemoryPanel();
```

Comment:
This doesn't look like it is used in this class. Only WinAppDriver code should be in this class.

Reply:
Change will be in my next commit